weatherman.py

```diff
@@ -0,0 +1,246 @@
+__weather__ = {}  # for storing weather records
+__months__ = {1: "Jan", 2: "Feb", 3: "Mar", 4: "Apr", 5: "May", 6: "Jun", 7: "Jul", 8: "Aug", 9: "Sep", 10: "Oct",
+              11: "Nov", 12: "Dec"}
+
+import os
+import sys
+
+
+class Parser:
+
+    @staticmethod
+    def extracting_info_file_name(file_name):
+        city, weather, year, month = file_name.split("_")
+        city = city.split("/")[-1]
+        month, ext = month.split(".")
+        return city, year, month
+
+    def read_file(self, file_name):
```

Comment:
Function and variable names should be descriptive and intent revealing. Which files are being read? Are you only reading in this function? Rethink of this and revisit all function/method and variable names.